chainer/functions/array/moveaxis.py

```diff
@@ -0,0 +1,121 @@
+import six
+
+from chainer import cuda
+from chainer import function_node
+from chainer.utils import type_check
+
+
+def _normalize_axis_tuple(axis, ndim, xp):
+    if xp.isscalar(axis):
+        axis = (axis,)
+
+    ret = []
+    for ax in axis:
+        ret.append(ax % ndim)
+    return ret
+
+
+def _moveaxis(a, source, destination, xp):
+    if hasattr(xp, 'moveaxis'):
+        return xp.moveaxis(a, source, destination)
+    if isinstance(source, int):
```

Comment:
We may assume axes are tuples since they are checked in `Moveaxis.__init__`.

Reply:
I found numpy allows type difference between source and destination axis if these length is one.

ex. numpy.moveaxis(a, 0, (0)).

So I fixed specification and implementation.
Any problem?